tests/integration/tier0/non-destructive/custom-repository/test_custom_repository.py

```diff
@@ -37,7 +37,8 @@ def test_custom_valid_repo_without_rhsm(convert2rhel, set_custom_repository_file
     assert c2r.exitstatus == 1
 
 
-def test_custom_invalid_repo_without_rhsm(shell, convert2rhel):
+@pytest.mark.parametrize("environment_variables", ["CONVERT2RHEL_INCOMPLETE_ROLLBACK"], indirect=True)
+def test_custom_invalid_repo_without_rhsm(shell, convert2rhel, environment_variables):
```

Comment:
it seems that we have [fixed](https://artifacts.osci.redhat.com/testing-farm/44229a03-daad-42bd-9ddc-107a6b099f53/work-non-destructive8fu4hf9h/plans/tier0/core/non-destructive/execute/data/guest/default-0/tests/integration/tier0/non-destructive/custom-repository/custom_invalid_repo_provided-14/output.txt) the test but since we introduced the INCOMPLETE_ROLLBACK env var and the rollback actually fails then the exit code for this test case is actually 1 and not 2

Reply:
Please, revert this.
It actually discovered a regression introduced in #1288  as Adam mentioned.
It's reported here https://issues.redhat.com/browse/RHELC-1753.
I don't really like that we're trying to make the test pass just for the sake of making it green :(

Reply:
Done. Should I trigger the whole suite again when the rpms are built? @danmyway

Reply:
Sorry @hosekadam, I missed the message, triggered the suite instead